js/client-teambuilder.js

```diff
@@ -1245,6 +1254,39 @@
 			this.save();
 		},
 
+		setNotesBlur: function (e) {
+			var textarea = $(e.currentTarget);
+			if (!$.trim(textarea.val())) textarea.parent().hide();
+			if (textarea.val().indexOf('|') >= 0 || textarea.val().indexOf(']') >= 0 || textarea.val().indexOf('\t') >= 0) {
+				app.addPopupMessage("Unfortunately, set notes can't contain tabs or the characters | and ], since they're used for storing teams.");
```

Comment:
Anyway, I did find info on persistent storage, and enabled it in b3b2eea2f21

Perhaps we should only support set notes if persistent storage is on?